panoramix/assets/javascripts/dashboard.js

```diff
@@ -45,10 +57,24 @@ var Dashboard = function (dashboardData) {
         }
       });
     },
-    clearFilter: function (slice_id) {
+    clearFilters: function(slice_id) {
       delete this.filters[slice_id];
       this.refreshExcept(slice_id);
     },
+    removeFilter: function(slice_id, col, vals) {
+      if (slice_id in this.filters) {
+        if (col in this.filters[slice_id]) {
+          var a = [];
```

Comment:
since these methods lack comments, you could improve readability a lot with better var naming here.